doc/architecture/README.md

```diff
@@ -0,0 +1 @@
+# architecture
```

Comment:
Should all be capitalized. Please also write a bit about it, e.g. that we use arc42, have a frontend+backend etc.

`1.5. ## Solution` there is some problem, please check the titles.

```suggestion
# Architecture
```